src/viewer.js

```diff
@@ -668,154 +850,144 @@ class VolumeImageViewer {
+  /** Gets the channel visualization/presentation parameters given an id
+   * @param {string} opticalPathID of the channel
+   * @param {number[]} color
+   * @param {number} opacity
+   * @param {number[]} thresholdValues
+   * @param {boolean} visible
+   */
+  getOpticalPathBlendingInformation(opticalPathID) {
+    const channel = this.getOpticalPath(opticalPathID)
+    if (channel === null) {
+      return null;
+    }
 
-    this[_interactions] = {
-      draw: undefined,
-      select: undefined,
-      modify: undefined
-    };
+    return channel.getBlendingInformation();
+  }
 
-    this[_controls] = {
-      scale: new ScaleLine({
-        units: 'metric',
-        className: ''
-      })
+  /** Adds the channel to the OpenLayer Map given an id
+   * @param {string} opticalPathID of the channel
+   */
+   activateOpticalPath(opticalPathID) {
+    const channel = this.getOpticalPath(opticalPathID)
+    if (channel === null) {
+      return;
     }
-    if (options.controls.has('fullscreen')) {
-      this[_controls].fullscreen = new FullScreen();
+
+    if (this.isChannelInOpenLayerMap(channel)) {
+      return;
     }
-    if (options.controls.has('overview')) {
-      const overviewImageLayer = new TileLayer({
-        extent: extent,
-        source: rasterSource,
-        preload: 0,
-        projection: projection
-      });
 
-      const overviewView = new View({
-        projection: projection,
-        resolutions: resolutions,
-        rotation: rotation
-      });
+    channel.blendingInformation.addToMap = true;
+    // NOTE: _drawingLayer has to be the last layer, otherwise the compistion will be broken
+    this[_map].removeLayer(this[_drawingLayer])
+    this[_map].addLayer(channel.tileLayer)
+    this[_map].addLayer(this[_drawingLayer])
+  }
 
-      this[_controls].overview = new OverviewMap({
-        view: overviewView,
-        layers: [overviewImageLayer],
-        collapsed: false,
-        collapsible: false,
-      });
+  /** Removes the channel to the OpenLayer Map given an id
+   * @param {string} opticalPathID of the channel
+   */
+   deactivateOpticalPath(opticalPathID) {
+    const channel = this.getOpticalPath(opticalPathID)
+    if (channel === null) {
+      return;
     }
 
-    /** Creates the map with the defined layers and view and renders it. */
-    this[_map] = new Map({
-      layers: [this[_imageLayer], this[_drawingLayer]],
-      view: view,
-      controls: [],
-      keyboardEventTarget: document,
-    });
+    if (!this.isChannelInOpenLayerMap(channel)) {
+      return;
+    }
 
-    this[_map].addInteraction(new MouseWheelZoom());
+    channel.blendingInformation.addToMap = false;
+    this[_map].removeLayer(channel.tileLayer)
+  }
 
-    for (let control in this[_controls]) {
-      this[_map].addControl(this[_controls][control]);
+  /** Returns if the channel is being rendered
+   * @param {object} channel
+   * @type {boolean} visible
+   */
+  isChannelInOpenLayerMap(channel) {
```

Comment:
I suggest renaming this method to `isOpticalPathActive()`, which would better fit to `activateOpticalPath()` and `deactivateOpticalPath()`.